Sources/NetworkProtection/Subscription/NetworkProtectionKeychainTokenStore+SubscriptionTokenHandling.swift

```diff
@@ -0,0 +1,45 @@
+//
+//  NetworkProtectionKeychainTokenStore+SubscriptionTokenHandling.swift
+//  DuckDuckGo
```

Comment:
This SwiftLint error can be fixed by removing this line I think

Reply:
fixed, thanks